content/departments/product-engineering/process/cross-org-team-collab.md

```diff
@@ -0,0 +1,19 @@
+# Cross-org/team collaboration
+
+Sometimes, ideas (and even code) for something your team owns comes from other teams, orgs, or even departments at Sourcegraph. That's a good thing because everyone can contribute to the innovation and success of the product we all use, and the more we do of that sort of thing, the more of a flywheel we're generating for improvements and is one of the real advantages of being a software development company that makes products for software developers.
+
+That said, if the collaboration part of this is not handled well, you can end up in situations where we create duplicative product features, or waste time building things that eventually will need to be reworked, or even shut down.
+
+To avoid this, as an open core company we can look to open source to see how this kind of situation is handled:
+
+1. There should always be a clear Product & Engineering team `maintainer` who makes the final decision of whether to accept a suggestion/code from another team. You can refer to the [ownership chart](../engineering/process/engineering_ownership.md) to determine who the maintainer is, and they are the first person you should go to to discuss your idea.
+2. Similarly, if you're submitting code, they own the merge decision for changes to areas they maintain.
+3. When it comes to go-to-market, they also determine pricing and how it fits in with their product area.
+
+The earlier you engage with the maintainer the more likely you are to be successful, and avoid a situation where you've gotten ahead of yourself and might need to redo work: unlike open source, there is no forking the project to make your own version. In situations where the ownership is unclear, it might represent a new category, or things are otherwise fuzzy, you should reach out to #prod-eng-directors for help.
```

Comment:
```suggestion
The earlier you engage with the maintainer the more likely you are to avoid a situation where you've gotten ahead of yourself and might need to redo work: unlike open source, there is no forking the project to make your own version. In situations where the ownership is unclear, it might represent a new category, or things are otherwise fuzzy, you should reach out to #prod-eng-directors for help.
```